src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle implements Area, Draw {
+    private int radius;
+    private String color;
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void getCircle(int radius, String color) {
```

Comment:
```suggestion
    public Circle(int radius, String color) {
```
для цього краще використовувати конструктор, тоді ти зможеш створювати об'єкти через 
`Circle myCircle = new Circle(10, "blue");